src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/OverlappingTrainingDateRequestController.cs

```diff
@@ -57,5 +58,20 @@ public async Task<IActionResult> ValidateUlnOverlapOnStartDate(long providerId,
                 HasStartDateOverlap = result.HasStartDateOverlap
             });
         }
+
+        [HttpGet]
+        [Route("{apprenticeshipId}/getOverlapRequest")]
+        public async Task<IActionResult> GetRequest(long apprenticeshipId)
```

Comment:
I think the variable name should be draftApprenticeshipId

Reply:
updated